cmd/sealer/cmd/cluster/run.go

```diff
@@ -115,6 +118,24 @@ func NewRunCmd() *cobra.Command {
 					app, extension, nil, imageEngine, runFlags.Mode)
 			}
 
+			if exist := osi.IsFileExist(common.GetDefaultClusterfile()); exist {
+				file, err := os.Open(common.GetDefaultClusterfile())
+				if err != nil {
+					return err
+				}
+				context, err := ioutil.ReadAll(file)
+				if err != nil {
+					return err
+				}
+
+				if ok := strings.Contains(string(context), args[0]); !ok {
```

Comment:
No need check if cluster image same, if not same, the user also can't run cluster.

Reply:
Uer API to marshal it.